bin/mv.js

```diff
@@ -0,0 +1,239 @@
+        })
+        .catch(function (err) {
+            throw Cli.error.cancelled('Failed to create webtask. ' + err);
+        });
+}
+
+function loadProfile(name) {
+    debug('loadProfile: name=%s', name);
+
+    let config = new ConfigFile();
+
+    return config.getProfile(name);
+}
+
+function moveCronJob(profile, name, target, options) {
+    debug('moveCronJob: profile=%j, name=%s, target=%j, options=%j',
+        _.omit(profile, 'token'), name, target, options);
+
+    return coroutine(function*() {
+        let job;
+
+        try {
+            job = yield profile.getCronJob({
+                name: name
+            });
+        } catch (err) {
+            if (err.statusCode === 404) {
+                return;
+            }
+            throw err;
+        }
+
+        let webtask = yield profile.getWebtask({
+            name: target.name,
+            container: target.container
+        });
+
+        if (_.get(options, 'verify') && job.token !== options.verify) {
+            console.log(chalk.bold('* Warning: failed to verify the cron job token (no match).'));
+        }
+
+        yield target.profile.createCronJob({
+            name: target.name,
+            container: target.container,
+            token: webtask.token,
+            schedule: job.schedule
+        });
+
+        yield profile.removeCronJob({
+            name: name
+        });
+    })();
+}
+
+function copyStorage(webtask, target) {
+    debug('copyStorage:webtask=%j, target=%j',
+        _.omit(webtask, 'token'), target);
+
+    return coroutine(function*() {
+        let body = yield exportStorage(webtask);
+        let data = _.get(body, 'data');
+        if (!_.isEmpty(data)) {
+            yield importStorage(target, {data});
+        }
+    })();
+}
+
+function exportStorage(webtask) {
+    debug('exportStorage: webtask=%j',
+        _.omit(webtask, 'token'));
+
+    return coroutine(function*() {
+        let url = `${webtask.sandbox.url}/api/webtask/${webtask.container}/${webtask.claims.jtn}/data`;
+        let res = yield request.get(url).set('Authorization', `Bearer ${webtask.sandbox.token}`);
+
+        debug('exportStorage: res.body=%j', res.body);
+
+        return res.body;
+    })();
+}
+
+function importStorage(webtask, data) {
+    debug('importStorage: webtask=%j, data=%j',
+        _.omit(webtask, 'token'), data);
+
+    return coroutine(function*() {
+        let url = `${webtask.profile.url}/api/webtask/${webtask.container}/${webtask.name}/data`;
+        let res = yield request.put(url)
+            .set('Authorization', `Bearer ${webtask.profile.token}`)
+            .send(data);
+
+        debug('importStorage: res.body=%j', res.body);
+
+        return res.body;
+    })();
+}
+
+function cloneWebtaskData(data) {
+    let clone = _(_.clone(data)).omit(['jti', 'iat', 'ca', 'webtask_url']).value();
+    if (url.parse(data.url).protocol === 'webtask:') {
+        delete clone.url;
+    } else {
+        delete clone.code;
```

Comment:
Trying to think out loud... If we have a webtask that has a url claim pointing to `http(s)://`, when you inspect it, will the result contain both `url` and `code` elements? 

We want to make sure that the new copy of the webtasks will preserve the `http(s)://...` url claims as opposed to taking a snapshot of the code at the time of copy.